build.gradle

```diff
@@ -135,7 +146,9 @@ subprojects {
             compile deps.log4j
             compile deps.guice
             compile deps.slf4j
-
+            // See: https://github.com/rzwitserloot/lombok/issues/1301
```

Comment:
Does this change require any additional change to the IDE to support lombok?

Reply:
It's used in gobblin-metrics, I did not try using a lombok annotation directly in my class. I could run azkaban tests without anything extra.